engine/render/src/render/font_renderer.cpp

```diff
@@ -1081,6 +1092,21 @@ namespace dmRender
         }
     }
 
+    static void RenderListFrustumCulling(dmRender::RenderListVisibilityParams const &params)
+    {
+        DM_PROFILE("FrustumCulling"); // TODO - this is the same as in Sprites. Maybe change it ?
```

Comment:
The comment seems more like a PR comment.

Agreed, currently, the dmRender::FrustumCulling function sets the same name.
I think each implementation (sprite, label etc) should then set it's own name: `DM_PROFILE("Label");`

Please go ahead and change this.

The profile scopes are hierarchical, so it shouldn't be a problem.
And, if they're in the

Reply:
Ok, i'll update both label and sprite RenderListFrustumCulling() methods then with respective log statements.

> The comment seems more like a PR comment.
yes. Clearly something to settle while reviewing.